packages/ember-simple-auth/addon/services/session.js

```diff
@@ -1,4 +1,4 @@
-import { alias, oneWay } from '@ember/object/computed';
+import { alias, readOnly } from '@ember/object/computed';
```

Comment:
This seems unrelated?

Reply:
There are 2 unit tests which expect three properties in session service to be readOnly and fail when set. So I have changed properties isAuthenticated, data and store to readOnly.

Reply:
But `oneWay` should be the same? My point is migrating the tests from Mocha to Qunit shouldn't require any changes in the tested code?

Reply:
oneWay will not throw an error when set. I have found the bug in mocha tests so the bug was always there.

```
it('is read-only', function() {
      expect(() => {
        sessionService.set('store', 'some other store');
      }).to.throw;
    });
```
it should be
```
}).to.throw();
```
Change is authenticated from oneWay to readOnly has fixed 3 tests, but it can be considered as breaking change. So maybe we should remove these tests?

Reply:
Hm, we should definitely move that into a separate PR to not also mix actual changes to the behavior into this – it's already a huge PR (necessarily) so I'd keep it as limited as possible :)